control.go

```diff
@@ -130,6 +130,7 @@ func (c *controlConn) shuffleDial(endpoints []string) (conn *Conn, err error) {
 		}
 
 		hostInfo, _ := c.session.ring.addHostIfMissing(host)
+		// TODO (gus) use translated address here?
```

Comment:
No, we don't. That was for me as I was looking around and forgot to clean the comment out. Thanks.